tensorboard/uploader/proto/blob.proto

```diff
@@ -12,3 +12,14 @@ enum BlobState {
   // Object is finalized.
   CURRENT = 2;
 }
+
+message Blob {
```

Comment:
Blob/BlobSequence seem very minimal here, is the plan to add the other fields as you work through the implementation?

Reply:
They are minimal, but they satisfy the need of the takeout task.

Other fields can be added to these message types in the future when the need arises.